uniswap-v2/proto/v1/uniswap.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+import "erc20.proto";
+package messari.uniswap.v2;
+
+message Pools {
+  repeated Pool pools = 1;
+}
+
+message Pool {
+  string name = 1;
+  string symbol = 2;
+  string address = 3;
+
+  messari.erc20.v1.ERC20Tokens input_tokens = 4;
+  messari.erc20.v1.ERC20Token output_token = 5;
+
+  int64 created_timestamp = 6;
+  int64 created_block_number = 7;
+}
+
+message Events {
+  repeated Event events = 1;
+}
+
+message Event {
+  oneof type {
+    DepositEvent deposit_type = 10;
+    WithdrawEvent withdraw_type = 20;
+    SyncEvent sync_type = 30;
+    SwapEvent swap_type = 40;
+  }
+
+  string hash = 100;
+  uint32 log_index = 101;
+  uint64 log_ordinal = 102;
+  string to = 103;
+  string from = 104;
+  uint64 block_number = 105;
+  uint64 timestamp = 106;
+  string pool = 107;
+}
+
+message DepositEvent {
+  repeated string input_token_amounts = 1;
+  optional string output_token_amount = 2;
+}
+
+message WithdrawEvent {
+  repeated string input_token_amounts = 1;
+  optional string output_token_amount = 2;
+}
+
+message SyncEvent {
+  string reserve0 = 1;
```

Comment:
@jaimehgb Didn't Alex say that the substreams CLI now has support for debugging BigInts? Shall we switch back to BigInt types now?